docs/_interface/Command-Line-Options.md

```diff
@@ -57,11 +57,8 @@ ETHEREUM OPTIONS:
   --lightkdf                          Reduce key-derivation RAM & CPU usage at some expense of KDF strength
   --eth.requiredblocks value          Comma separated block number-to-hash mappings to require for peering (<number>=<hash>)
   --mainnet                           Ethereum mainnet
-  --ropsten                           Ropsten network: pre-configured proof-of-stake test network
-  --rinkeby                           Rinkeby network: pre-configured proof-of-authority test network
```

Comment:
> This file is the output of geth --help, no point in changing it. It will eventually follow the updates we make to the CLI.

That goes for this change too